test/images/agnhost/README.md

```diff
@@ -124,6 +142,11 @@ Usage:
 ```
 
 
+### inclusterclient
+
+TBA
```

Comment:
Same for here --- would be nice to add a blurb describing what this does.

Reply:
yep, which is why I've added the TBA. I've tried to figure it out from the code, but for this, at a first glance, I didn't quite make out what it's supposed to do. Will amend later, after a bit of digging.